README.md

```diff
@@ -16,4 +16,10 @@
 
 ---
 
-A PostgreSQL extension providing an async networking interface accessible via SQL using a [background worker](https://www.postgresql.org/docs/current/bgworker.html) and curl.
+pg_net is a PostgreSQL extension exposing a SQL interface for async networking with a focus on scalability and UX.
+
+Features:
+
+- async http GET requests
+- async http POST requests with a JSON payload
+- +300 requests/sec
```

Comment:
Did you test this one? Perhaps I should do a LIMIT 300 to the number of rows(and thus opened sockets) that are read from the `http_request_queue`?

Reply:
I tested 1000 and it finished in < 1 second.

It still has weird performance characteristics (slow) if you exceed 1000

Eh, you're right, I'll remove the performance claims